packages/generator/src/generator.js

```diff
@@ -94,6 +101,10 @@ export default class Generator {
     return routes
   }
 
+  getAppRoutes () {
+    return require(path.join(this.options.buildDir, 'router/routes.json'))
```

Comment:
Isn't a `fs.exists` guard necessary?

Reply:
Well actually the guard is done in `initiate` directly :)